### PR TITLE
BUILD: Automatically add the tag for the version (if not a debug build).

### PR DESCRIPTION
**Pull request by blckmn**
To make the creation of the assets when doing release candidates simple.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by sugaarK — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
